code/modules/research/designs/power_designs.dm

```diff
@@ -217,3 +217,68 @@
 		RND_CATEGORY_STOCK_PARTS + RND_SUBCATEGORY_STOCK_PARTS_TURBINE
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING
+
+/datum/design/diode_disk_stamina
+	name = "Electrodisruptive Diode Disk"
+	desc = "A stamina damaging and SM healing Diode Disk."
```

Comment:
```suggestion
	desc = "A stamina damaging and supermatter crystal healing Diode Disk."
```
Actually spell it out. Don't use abbreviations for descriptive text like this. Do this for all the other instances of SM.